test/built-ins/BigInt/asIntN/bigint-tobigint-errors.js

```diff
@@ -11,6 +11,14 @@ features: [BigInt, computed-property-names, Symbol, Symbol.toPrimitive]
 ---*/
 assert.sameValue(typeof BigInt, 'function');
 assert.sameValue(typeof BigInt.asIntN, 'function');
+
+assert.throws(TypeError, function () {
+  BigInt.asIntN();
+}, "ToBigInt: no argument => undefined => TypeError");
+assert.throws(TypeError, function () {
+  BigInt.asIntN(0);
+}, "ToBigInt: no argument => undefined => TypeError");
```

Comment:
Is this intentional?

Reply:
yes, it's due to asIntN using the default behavior for missing arguments, combined with ToBigInt(undefined) throwing an exception (ToBigInt only does implicit conversions for booleans and strings)